cypress/e2e/GUI/listClients.cy.js

```diff
@@ -0,0 +1,91 @@
+const tableClients = 'table'
+const tableClientsHead = 'table tbody tr'
+const tableClientsCels = 'table tbody tr:first-child td'
+const loadingLabel = '#loading'
+const imageNoCustomersAvaliable = 'div[style="display: flex; justify-content: center; align-items: center; height: 100%;"]'
+const noCustomersInfoText = '.no-customers-available-text'
+const inputName = '[data-testid="name"]'
+
+
+
+describe('Lista de Clientes', () => {
+
+    beforeEach(() => {
+      cy.visit('/')
+    });
+
+
+
+    it('Verificar se e exibido uma lista de clientes quando há dados no banco de dados', () => {
+
+        cy.get(tableClients, { timeout: 30000 }).should('be.visible')
+        cy.get(tableClientsHead).should('have.length.greaterThan', 0)
+        cy.get(tableClientsCels).should('have.length.greaterThan', 0)
+
+    });
+
+
+
+    it('Verificar se e exibido o elemento fallback de "Loading..." antes da busca inicial dos clientes', () => {
+       
+        cy.intercept('GET', '**/customers*').as('getCustomers')
```

Comment:
Para testar estados transitórios, como o componente de loading..., é importanto forçar o `delay` na resposta do servidor, conforme explico no blog post [Como simular a demora em uma requisição com Cypress](https://talkingabouttesting.com/2021/02/26/como-simular-a-demora-em-uma-requisicao-com-cypress/).